VirtualTest/Test2/Test2.ino

```diff
@@ -0,0 +1,360 @@
+#include "Tile.h"
+#include "TVector.h"
+#include "Map.h"
+
+uint8_t iSize = 11, iRow = 1, iColumn = 1;
+char virtualMap[11][11] = {{'.','W','.','W','.','W','.','W','.','W','.'},
+                    {'W','R','W',' ','W',' ','.',' ','.',' ','W'},
+                    {'.','.','.','.','.','.','.','.','.','.','.'},
+                    {'W',' ','.',' ','W',' ','W',' ','W',' ','W'},
+                    {'.','.','.','.','.','.','.','.','.','.','.'},
+                    {'W',' ','W',' ','W',' ','W',' ','W',' ','W'},
+                    {'.','.','.','.','.','W','.','.','.','.','.'},
+                    {'W',' ','.',' ','W',' ','W',' ','W',' ','W'},
+                    {'.','.','.','.','.','.','.','.','.','.','.'},
+                    {'W',' ','W',' ','.',' ','.',' ','.',' ','W'},
+                    {'.','W','.','W','.','W','.','W','.','W','.'}};
+
+void printVirtualMap() {
+  for(int iX = 0 ; iX < iSize ; iX++) {
+    for(int iY = 0 ; iY < iSize ; iY++) {
```

Comment:
if iSize is always the same, better name it something like length or something like that.